nilearn/maskers/tests/test_nifti_masker.py

```diff
@@ -604,7 +604,8 @@ def test_nifti_masker_io_shapes(rng):
 
 
 @pytest.mark.skipif(
-    have_mpl, reason="Test requires matplotlib not to be installed."
+    is_matplotlib_installed(),
+    reason="Test requires matplotlib not to be installed.",
```

Comment:
This marker repeats in many places. One option might be to define it in a module and import it from others.

Reply:
I count 4 repeats in total in the whole codebase: I think this is still manageable at this stage.